internal/translator/translate.go

```diff
@@ -109,15 +119,40 @@ func translateRoute(irRoute irv1beta1.Route) (hpv1.Route, []string) {
 	return hpRoute, warnings
 }
 
-func translateRoutes(irRoutes []irv1beta1.Route) ([]hpv1.Route, []string) {
+func translateInclude(irRoute irv1beta1.Route) *hpv1.Include {
+	var hpInclude *hpv1.Include
+
+	if irRoute.Delegate != nil {
```

Comment:
this could be rewritten as

if ir.Route.Delegate == nil {
    return nil
}

return &hpv1.Include{ ... }